test/options_test.cc

```diff
@@ -133,6 +135,11 @@ TEST_F(OptionsImplTest, All) {
   EXPECT_EQ(cmd->jitter_uniform().nanos(), options->jitterUniform().count());
   EXPECT_EQ(cmd->experimental_h1_connection_reuse_strategy().value(),
             options->h1ConnectionReuseStrategy());
+  i = 0;
+  for (const auto& label : cmd->labels()) {
```

Comment:
This feels like we are re-implementing a matcher that might already exist. I am assuming we are using googletest under the hood, so we should be able to utilize the existing container matches, specifically `ElementsAre()`.

https://github.com/google/googletest/blob/master/googlemock/docs/cook_book.md#matching-containers

That way we can directly compare two containers, the actual and the expected.

Reply:
Nice; done via `ElementsAreArray()` in 3cb9f42354387ed6bca47bb241c87b64087a0dc7